src/charts/donut/donut-pct-chart-directive.js

```diff
@@ -302,6 +310,29 @@ angular.module('patternfly.charts').directive('pfDonutPctChart', function (c3Cha
           };
         };
 
+        $scope.getCenterLabelText = function (scope) {
+          var centerLabelText;
+
+          // default to 'used' info.
+          centerLabelText = { bigText: scope.data.used,
+                              smText:  scope.config.units + ' Used' };
+
+          if (scope.config.centerLabelFn) {
+            centerLabelText.bigText = scope.config.centerLabelFn(scope);
```

Comment:
that would mean we could never change things in this scope for fear of how it could be used outside. The total, used, and available (calculated at times) are all set on the data object which is passed in along with the function to call, therefore that data should already be accessible to them.